README.md

```diff
@@ -43,13 +50,20 @@ your_profile_name:
   outputs:
     dev:
       type: databricks
+      catalog: [optional catalog name, if you are using Unity Catalog]
```

Comment:
Shall we put a note that this is available with `dbt-databricks>=1.1.1`?

Reply:
Done.